### PR TITLE
Moved res files to resources directory

### LINKED ISSUE #390: Cleanup: Remove `res` Folder
- Move `res/rococo-local.json` to new `resources` dir
- We want to keep the generated files separate from the spec files we want to save
- Check to see if we need to update `rococo-local.json`
- Once that is done:
    - Move the needed parts of res/genesis/readme to the root readme
    - delete `res`
    - add `res` to the gitignore
    - update generate spec scripts to mkdir as needed.
- Update [Directory Structure Wiki page](https://github.com/LibertyDSNP/frequency/wiki/Directory-Structure) about folders (don't remove res, just note that it is ignored and add the new one if any)
- Open to also renaming `res` for clarity

### PR DESCRIPTION
**Pull request by dopatraman**
# Goal
The goal of this PR is separate vital resource files from generated res files.

Closes #390 

# Discussion
There are empty README files within the `genesis/local`, `genesis/mainnet`, and `genesis/testnet` folders. Those have not been moved over because they didn't seem useful. Please let me know if they should be.

### CODE REVIEW

File: resources/rococo-local.json [hunk 1]
```diff
@@ -6,11 +6,10 @@
   "telemetryEndpoints": null,
   "protocolId": "dot",
   "properties": null,
-  "forkBlocks": null,
-  "badBlocks": null,
-  "lightSyncState": null,
   "consensusEngine": null,
   "codeSubstitutes": {},
+  "relay_chain": "rococo-local",
+  "para_id": 2000,
```
Comment (wilwade) [inline resources/rococo-local.json:12]: This is the spec for the relay chain, not a parachain chain, so it shouldn't have these two fields.

Not sure why it was trying to require them.

**Review by wilwade — CHANGES_REQUESTED** [review]
The genesis/README.md file should just be dropped I think. It is out of date information anyway and the root readme has all the info now.

File: resources/rococo-local.json (hunk 1 above)
Comment (dopatraman) [reply resources/rococo-local.json:12]: the build actually throws an error if they're not included.

**Review by shannonwells — APPROVED** [review]
I don't have anything to add. I'm not sure what the side-effects of the rococo-local chain spec are but since they're required (won't build otherwise) and it's for the local chain, I don't really see a problem.

**Review by saraswatpuneet — CHANGES_REQUESTED** [review]
Putting a hold for https://github.com/LibertyDSNP/frequency/pull/437

above PR has major changes which changes these files too, given we are running extensive test to ensure our features are working, this should go after 432

File: resources/rococo-local.json (hunk 1 above)
Comment (wilwade) [reply resources/rococo-local.json:12]: @dopatraman Which build fails for you? I pulled this branch and `make build`, `make lint`, and `make start-frequency` all work.

Comment (saraswatpuneet) [reply resources/rococo-local.json:12]: @dopatraman  I dont think this is needed, its not used in any builds

**Comment by saraswatpuneet** [comment]
@dopatraman  few things to try before merge

1. Run frequency locally to confirm it is working
2. Can you check where ./res/genesis folder is consume while generating specs, that might need an update too especially frequency_spec.yml job because make specs-rococo-2000/-4044/-mainnet dumps output in res/genesis/folders, please run those checks and update accordingly

**Comment by dopatraman** [comment]
@saraswatpuneet to my knowledge all the files consuming `rococo-local.json` have been updated. Any files that consume other generated `res/genesis` files should still be able to (we are still generating and writing files to that location, they are just not being committed to version control). Please let me know if I'm understanding this incorrectly.

**Review by wilwade — APPROVED** [review]
Really like the new `make specs-rococo-local`

**Review by enddynayn — APPROVED** [review]
fabulous!

**Review by saraswatpuneet — APPROVED** [review]
Ok cool then this won't break.the jobs

👍

**Pull request merged**